device-types/Fortinet/FG-401E.yaml

```diff
@@ -0,0 +1,93 @@
+---
+manufacturer: Fortinet
+model: FortiGate 401E
+slug: fortinet-fg-401e
+part_number: FG-401E
+u_height: 1
+weight: 16.9
+weight_unit: lb
+is_full_depth: false
+airflow: side-to-rear
+comments: Same as FG-400E, but **WITH** 2 x 240GB SSD onboard storage. [FortiGate 400E Series Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/FortiGate_400E.pdf)
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 221
+    allocated_draw: 115
+  - name: PS2
+    type: iec-60320-c14
+    maximum_draw: 221
+    allocated_draw: 115
```

Comment:
Thank you, you're correct -- the PSUs are modular. We already happen to carry the requisite module type definition in `Fortinet/SP-FG300E-PS.yaml`. Replaced the `power-ports` sections with `module-bays` ones matching the template name in SP-FG300E-PS.

Since this way we unfortunately lose max and allocated power draw data, I added that information to the device comment (it's not the same for 400E vs. 401E despite the PSU being the same). Hope that's alright this way!